src/Resources/public/Admin.js

```diff
@@ -758,6 +758,7 @@ var Admin = {
             window.history.pushState({
                 path: newurl
             }, '', newurl);
+            jQuery(this).closest('form').find('input[name="_tab"]').val(tab);
```

Comment:
is `closest` really the best option?

Reply:
As an alternative, we can add an id to the form and data attribute to the tabs.

Reply:
ping @kunicmarko20

Reply:
that sounds like a better way, not sure how reliable is `closest` what if there are multiple forms

Reply:
@kunicmarko20 @osavchenko please resolve this conversation.

Reply:
@kunicmarko20 I have changed behaviour in [new commit](https://github.com/sonata-project/SonataAdminBundle/pull/5314/commits/65e88ebf6df249cfa2b29f46ee343c4162b54bd7)